utils/build_pydusa_numpy.sh

```diff
@@ -17,11 +20,15 @@ set -xe
 
 RECIPES_DIR=$(cd $(dirname $0)/../recipes && pwd -P)
 
-if [ ${#args} -lt 1 ];then
-    numpy_versions=( 1.5 1.6 1.7 1.8 1.9 1.10 1.11 1.12 )
-else
-    numpy_versions=${args[@]}
-fi
+case ${#args[@]} in
+    1|2) numpy_versions=${args[0]}
+        
+        export GIT_PYDUSA_BRANCH=${2:-"v20170831"}
```

Comment:
I, still, think the default should be `"master"`. Developers can branch off and play in their branches, but master needs to be the stable branch. So, if the changes on `develop` on pydusa repo are the latest and greatest, it should be merged into `master`. Thus, the default here should be `"master"`.

Reply:
Yes you are right, but we do want the user to download the version that matches our latest stable release by default.

We plan to remove the develop branch soon and merge it into the master and when people want to try the daily snapshot, they can go for the master branch, that will be the latest and greatest.

Reply:
Then, every time you change the tag on pydusa repo, you will have to update it here in this script?